oh_staff_ui/views.py

```diff
@@ -106,9 +106,24 @@ def item_search(request: HttpRequest) -> HttpResponse:
                 typed_query = form.cleaned_data["status_query"]
             else:
                 typed_query = form.cleaned_data["char_query"]
+
+            # check for filter values, and set to known defaults if not present
+            item_type_filter = form.cleaned_data["item_type_filter"]
```

Comment:
These assignments & comparisons can be simplified.  Instead of reading `form.cleaned_data["some_filter"]` twice, this looks cleaner:
```
item_type_filter = form.cleaned_data["item_type_filter"]
if item_type_filter is None:
    item_type_filter = "all"
```
Could also be done with [ternaries](https://docs.python.org/3.10/reference/expressions.html#conditional-expressions) but I think that would be less readable here.